linux_os/guide/system/software/integrity/software-integrity/aide/aide_scan_notification/oval/shared.xml

```diff
@@ -17,7 +17,11 @@
   </ind:textfilecontent54_test>
   <ind:textfilecontent54_object comment="notify personnel when aide completes" id="object_test_aide_scan_notification" version="1">
     <ind:filepath>/etc/crontab</ind:filepath>
+    {{% if product == "sle12" %}}
+    <ind:pattern operation="pattern match">^.*/usr/bin/aide[\s]*\-\-check.*\|.*/bin/mail[\s]*-s[\s]*".*"[\s]*.+@.+$</ind:pattern>
+    {{% else %}}
     <ind:pattern operation="pattern match">^.*/usr/sbin/aide[\s]*\-\-check.*\|.*/bin/mail[\s]*-s[\s]*".*"[\s]*.+@.+$</ind:pattern>
+    {{% endif %}}
```

Comment:
The only change is the binary path, right?
How about defining and using a Jinja variable with the path?
Similar to what is done here:
https://github.com/ComplianceAsCode/content/blob/3ada19b6423cf589336e61d1f873b1efb2758308/linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_configure_remote_server/oval/shared.xml#L1-L5

Reply:
Thanks! I will go with that. That'll make the file easier to read.